src/components/UserFeed.jsx

```diff
@@ -1,22 +1,30 @@
 import { useAuth } from "./AuthContext";
+import { useNavigate, useParams } from "react-router-dom";
 import React, { useState, useEffect } from "react";
 import RecipeList from "./RecipeList";
-import { Container, Spinner, ButtonGroup, Button, Row } from "react-bootstrap";
+import { Container, Spinner, ButtonGroup, Button, Row, Modal } from "react-bootstrap";
 
 function UserFeed() {
   const [myUserName, setMyUserName] = useState(localStorage.getItem("currentUser"));
+  const navigate = useNavigate();
   const { token, logout, isLogged } = useAuth();
   const [myUserFollowing, setMyUserFollowing] = useState([]);
-  const [recipes, setRecipes] = useState([]);
-  const [FollowingRecipes, setFollowingRecipes] = useState([]);
+  const [recipesForYou, setRecipesForYou] = useState([]);
+  const [recipesFollowing, setRecipesFollowing] = useState([]);
+  const [followingRecipes, setFollowingRecipes] = useState([]);
+  const [logged, setLogged] = useState(false);
```

Comment:
is Auth logged not working properly?

Reply:
Not at the time I changed it, that's why I added that extra variable, everything and I can test it again but it comes from the problem with useAuth that we discussed in class.